var/spack/repos/builtin/packages/py-pyfftw/package.py

```diff
@@ -13,6 +13,7 @@ class PyPyfftw(PythonPackage):
     homepage = "http://hgomersall.github.com/pyFFTW"
     url      = "https://pypi.io/packages/source/p/pyFFTW/pyFFTW-0.10.4.tar.gz"
 
+    version('0.12.0', sha256='60988e823ca75808a26fd79d88dbae1de3699e72a293f812aa4534f8a0a58cb0')
     version('0.11.1', sha256='05ea28dede4c3aaaf5c66f56eb0f71849d0d50f5bc0f53ca0ffa69534af14926')
     version('0.10.4', sha256='739b436b7c0aeddf99a48749380260364d2dc027cf1d5f63dafb5f50068ede1a')
 
```

Comment:
A few additional changes I would recommend to this package:

1. The numpy dependency should be updated to `@1.10:1.999`
2. The cython dependency should be updated to `@0.29:0.999`
3. The scipy dependency is actually optional, I would either remove it or add a `scipy` variant that defaults to False

Reply:
Thank you for the super quick feedback! I will work on adding all those points.

Reply:
Oh wow, our reviews collided mid-air :)
See a detailed update below if you like.

Reply:
I was thinking that the `:` between versions functioned in a python-like way (for ranges of indexes).
But then, what does the `0.999` part represent?
Sorry for the ignorant question, but I am having a bit of trouble finding it in google...

Reply:
The thing behind the `:` is the inclusive upper version.
You don't need to technically add them in advance unless we know a project follows [semantic versioning](https://semver.org/) for unknown future releases.

Reply:
Thank you for the reply @ax3l , that makes sense.

Reply:
I think that now the version of the `packages.py` file is according to the suggestions. Thank you both @adamjstewart and @ax3l for your help and for explaining everything.